app/src/main/java/com/github/se/project/ui/profile/ProfileInfo.kt

```diff
@@ -50,12 +51,16 @@ fun ProfileInfoScreen(
             horizontalArrangement = Arrangement.SpaceBetween) {
               Row(Modifier.padding(horizontal = 4.dp)) {
                 IconButton(
-                    onClick = { /* Navigate to calendar screen */},
+                    onClick = { navigationActions.navigateTo(Screen.EDIT_SCHEDULE) },
```

Comment:
We can access the schedule edit screen as a Student which should not be the case